src/main/java/longah/node/Transaction.java

```diff
@@ -46,6 +50,15 @@ public Transaction(Member lender, ArrayList<Subtransaction> subtransactions,
             }
         }
 
+        if (transactionTime != null) {
```

Comment:
Do this as an overloaded method with the additional parameter that calls this method instead

Reply:
Resolved!